nibabel/tests/test_image_load_save.py

```diff
@@ -265,62 +263,3 @@ def test_filename_save():
             del rt_img
         finally:
             shutil.rmtree(pth)
-
-
-def test_analyze_detection():
```

Comment:
These tests (and the rest of the removed ones in this file) need to get similar tests for the current code structure.